content/article/introducing-dbix-class-resultset-prettyprint.md

```diff
@@ -0,0 +1,351 @@
+
+  {
+    "title"       : "Introducing DBIx::Class::ResultSet::PrettyPrint",
+    "authors"     : ["paul-cochrane"],
+    "date"        : "2025-06-03T14:03:48",
+    "tags"        : [],
+    "draft"       : false,
+    "image"       : "/images/introducing-dbix-class-resultset-prettyprint/dbic-resultset-prettyprint-synopsis-cover.png",
+    "thumbnail"   : "",
+    "description" : "",
```

Comment:
Could we get a thumbnail and a description as well? You can see on the home page that the articles with descriptions are a little less mysterious. 😄

Reply:
Sure, no probs. Thanks for letting me know about the missing fields. :-)